sqladmin/models.py

```diff
@@ -125,7 +125,18 @@ def is_accessible(self, request: Request) -> bool:
         return True
 
 
-class ModelAdmin(BaseModelAdmin, metaclass=ModelAdminMeta):
+class ModelView(BaseModelAdmin):
```

Comment:
Maybe this can be something like `View` or `BaseView`?
I think the name `Model...` is reflecting database model and it's not entirely correct in this case.

Reply:
yeah BaseView is more clear :)